src/chi/javascript/components/sidenav.js

```diff
@@ -98,6 +102,44 @@ class Sidenav extends Component {
     });
   }
 
+  _resetDrawerAnimation() {
+    this._drawers.forEach(function(drawer) {
+      if (!drawer._config.animated) {
+        drawer._config.animated = true;
+      }
+    });
+  }
+
+  _openDrawersOnHover() {
+    const disableAnimation = (drawer) => {
+      this._drawers.forEach((otherDrawer) => {
+        if (otherDrawer !== drawer) {
+          otherDrawer._config.animated = false;
+        }
+      });
+    };
+    const enableAnimation = () => {
+      this._drawers.forEach((otherDrawer) => {
+        otherDrawer._config.animated = true;
+      });
+    };
+    const _openDrawerOnMouseEnter = (drawer) => {
+      drawer.show();
+      disableAnimation(drawer);
+    };
+    const _closeDrawerOnMouseleave = (drawer) => {
```

Comment:
@janibolkvadze Hover behavior looks great. Click behavior is a little buggy - the drawer abruptly closes when any first tier menu item is clicked. Is the drawer not animating on close due to -active getting removed before -transitioning has completed?